### ISSUE TITLE
Update uif-table to render HTML table markup

### ISSUE THREAD

**Issue by andrewconnell** [ngOfficeUIFabric/ng-officeuifabric#294]
Fabric's 2.3.0 release added the ability to use HTML markup for building tables https://github.com/OfficeDev/Office-UI-Fabric/pull/367 instead of the original method of using `DIV` elements. This is better for accessibility and as such our table should be updated to use HTML table elements rather than DIV.

**Comment by waldekmastykarz** [comment]
Let me pick it up 😄
